test/e2e/runner/load.go

```diff
@@ -63,13 +63,13 @@ func Load(ctx context.Context, testnet *e2e.Testnet) error {
 			// should error; however, for very short tests
 			// we shouldn't abort.
 			//
-			// The 2s cut off, is a rough guess based on
-			// the expected value of
-			// loadGenerateWaitTime. If the implementation
-			// of that function changes, then this might
-			// also need to change without more
-			// refactoring.
-			if success == 0 && time.Since(started) > 2*time.Second {
+			// The 5s cut off, is a rough guess based on
+			// the expected value of loadGenerateWaitTime
+			// and typical throughput of the load
+			// generation. If these implementations
+			// changes, then this might also need to
+			// change without more refactoring.
```

Comment:
Wouldn't it be better if we just made the single node tests run a little longer (so that transactions do get through)?

Reply:
I've tried doing that, but because the load generator doesn't currently have a way to communicate the number of transaction's we've seen the best we can sleep a bit before canceling the context and *hope*.